### PR TITLE
Add `pendingTestCase` command and use it.

### PR DESCRIPTION
**Pull request by fisx**
This should make life slightly more bearable for everybody including concourse, while still allowing to run the pending tests locally by changing the definition to `pendingTestCase = testCase`.

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by fisx** [comment]
@akshaymankar I know you're opposed to this, but I don't think it's a sustainable situation that we have to run the test suite 2-3 times consistently *every time* we want to get a PR green.  With `pendingTestCase`, we can keep track of where we need to fix things, while not being in the way of everybody else.

Please please please say ok!

**Comment by akshaymankar** [comment]
What process do imagine around this? When we mark something as pending, how and when does it get addressed?

**Comment by fisx** [comment]
> What process do you imagine around this? When we mark something as pending, how and when does it get addressed?

same process as now, only we don't disrupt everybody else: somebody picks it up and fixes it as time permits.

i've also added a make rule to list pending tests, and a shell variable to run them anyway.  the important thing is that we don't set this shell variable on concourse.  maybe we can set it in a pipeline where we build release artefacts?

**Comment by fisx** [comment]
> maybe we can set it in a pipeline where we build release artefacts?

if you want to pair on this let me know.

**Review by elland — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -270,8 +275,9 @@ ifeq ($(INTEGRATION_FEDERATION_TESTS), 1)
 	$(EXE_SCHEMA) --keyspace $(package)_test2 --replication-factor 1 --reset
 endif
 endif
-	./dist/brig-index reset --elasticsearch-index directory_test --elasticsearch-server http://localhost:9200 > /dev/null
-	./dist/brig-index reset --elasticsearch-index directory_test2 --elasticsearch-server http://localhost:9200 > /dev/null
+	@echo "brig-index reset currently does not work, see https://github.com/wireapp/wire-server/pull/3114"
+#	./dist/brig-index reset --elasticsearch-index directory_test --elasticsearch-server http://localhost:9200 > /dev/null
+#	./dist/brig-index reset --elasticsearch-index directory_test2 --elasticsearch-server http://localhost:9200 > /dev/null
```
Comment (jschaul) [inline Makefile:280]: please rebase on develop; this change should not be necessary anymore.

**Comment by fisx** [comment]
integration tests pass locally.  i don't know why concourse doesn't run them on this PR.

**Pull request merged**